src/modules/endpoint/models/__tests__/CategoriesContainer.spec.js

```diff
@@ -0,0 +1,42 @@
+import CategoriesContainer from '../CategoriesContainer'
+import CategoryModel from '../CategoryModel'
+
+describe('CategoriesContainer', () => {
+  const categories = [
+    new CategoryModel({id: 0, url: '/augsburg/de', title: 'augsburg'}),
+    new CategoryModel({id: 20, url: '/augsburg/de/willkommen', parentId: 0, parentUrl: '/augsburg/de', title: 'willkommen'}),
+    new CategoryModel({id: 21, url: '/augsburg/de/erste-schritte', parentId: 0, parentUrl: '/augsburg/de', title: 'erste-schritte'}),
+    new CategoryModel({id: 22, url: '/augsburg/de/erste-schritte/asylantrag', parentId: 21, parentUrl: '/augsburg/de/erste-schritte', title: 'asylantrag'})
+  ]
+
+  const categoriesContainer = new CategoriesContainer(categories)
+
+  const category1 = categoriesContainer.getCategoryByUrl('/augsburg/de/')
```

Comment:
This should be in a separate test. You can not be sure here that category1 is the first CategoryModel here.
So test against the previsously defined test data.

Reply:
you're right, done